config/samples/meteor_v1alpha1_customnbimage.yaml

```diff
@@ -72,6 +72,31 @@ spec:
 ---
 apiVersion: meteor.zone/v1alpha1
 kind: CustomNBImage
+metadata:
+  name: s2i-minimal-py38-notebook-import-with-secret # TODO
```

Comment:
Like :+1: 

However, I also like having `ImageImport` samples in the separate file you are also adding, so I think this (duplicate) should be removed from here.

Maybe go one step further and separate each sample to its own file to make it easier to pick e.g. for demos? That can be a separate PR though - for now I would say let's not add duplicate content